hexl/CMakeLists.txt

```diff
@@ -119,45 +128,89 @@ else ()
     # Export logging only if we are debugging
     if (${HEXL_DEBUG})
         if (CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
-            add_custom_command(TARGET hexl POST_BUILD
-                COMMAND ar -x $<TARGET_FILE:hexl>
-                COMMAND ar -x $<TARGET_FILE:cpu_features>
-                COMMAND ar -x $<TARGET_FILE:gflags>
-                COMMAND ar -x $<TARGET_FILE:easyloggingpp>
-                COMMAND ar -qcs $<TARGET_FILE:hexl> *.o
-                COMMAND rm -f *.o
-                WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-                DEPENDS hexl cpu_features gflags easyloggingpp
+            if(CpuFeatures_FOUND)
+                add_custom_command(TARGET hexl POST_BUILD
```

Comment:
We'll probably want to extract this to a function at some point, but it's fine for now.